css/index.css

```diff
@@ -143,4 +177,149 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.box:nth-child(n) {
```

Comment:
We already talked about this but the cleanest way to do this would be to use the .box selector for everything that is the same for all boxes and then use nth child just for the things that are different between them.